app/views/users/_user.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class='user-page'>
+    <div class='users'>
+      <div class='img'>
+        <img  src="<%= user.photo %>" width="100" height="100" alt="Profile picture">
+      </div>
+      <div>
+        <div class='main'>
+          <h1 class='name'> <%= user.name %> </h1>
+          
+            <%= render layout: "shared/tag" do %>
+              <p class='counter'> Number of posts: <%= user.posts_counter %> </p>
+            <% end %>
+        
+        </div>
+      </div>
+    </div>
+</div>
```

Comment:
- [ ] Kindly add a link to click on each user take you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)